client/extensions/woocommerce/woocommerce-services/lib/utils/format-date.js

```diff
@@ -0,0 +1,11 @@
+export default ( date ) => {
+	// Note: Safari ignores the 'locale' and 'options' argument. It will display
+	// the full date (with timezone and everything) in the browser's locale (instead of WP lang)
+	return new Date( date ).toLocaleDateString( document.documentElement.lang, {
```

Comment:
Probably in Calypso there's a better way to get the user's locale. Or better yet, a way to format dates consistently. I think we added this logic to save us the weight of `moment`, but `wp-calypso` already has it.